gateway/auth/cbauthauthenticator.go

```diff
@@ -25,15 +25,17 @@ type authenticatorState struct {
 type CbAuthAuthenticator struct {
 	authenticatorState gocbcorex.AtomicPointer[authenticatorState]
 
-	logger   *zap.Logger
-	username string
-	password string
-	service  string
+	logger      *zap.Logger
+	username    string
+	password    string
+	service     string
+	clusterUUID string
```

Comment:
NIT: [rant]This is a prime example of why aligning declarations to look pretty is an anti-pattern. A single field was added, but the git history for the existing fields is now tainted with this change.[/rant]